niworkflows/viz/utils.py

```diff
@@ -222,19 +223,27 @@ def plot_segs(
     compress="auto",
     **plot_params
 ):
-    """ plot segmentation as contours over the image (e.g. anatomical).
+    """
+    Generate a static mosaic with ROIs represented by their delimiting contour.
+
+    Plot segmentation as contours over the image (e.g. anatomical).
     seg_niis should be a list of files. mask_nii helps determine the cut
     coordinates. plot_params will be passed on to nilearn plot_* functions. If
     seg_niis is a list of size one, it behaves as if it was plotting the mask.
     """
     plot_params = {} if plot_params is None else plot_params
 
-    image_nii = _3d_in_file(image_nii)
+    image_nii = as_canonical(_3d_in_file(image_nii))
+    seg_niis = [as_canonical(_3d_in_file(f)) for f in seg_niis]
     data = image_nii.get_fdata()
 
     plot_params = robust_set_limits(data, plot_params)
 
-    bbox_nii = nb.load(image_nii if bbox_nii is None else bbox_nii)
+    bbox_nii = (
+        image_nii if bbox_nii is None
+        else as_canonical(_3d_in_file(bbox_nii))
+    )
```

Comment:
I'm not entirely sure what bbox is supposed to be, so it's hard to say if reorienting it can be potentially problematic.

Reply:
bbox is a mask from which the visible extent of the plot will be calculated. Typically, you pass a brain mask and the mosaic will be zoomed in into the bounding box of that brain mask (thus, making sure you don't render blank planes).